### PR TITLE
Change Intake to use Victor SPX

### LINKED ISSUE #63: Change Intake to use Victor SPX motor controllers

### PR DESCRIPTION
**Pull request by kchenturtles**
Changed IdleMode to NeutralMode, deleted CANSparkMax, etc.

Closes #63

### CODE REVIEW

**Review by chit-sudeep — APPROVED** [review]
Reviewed all changes.

Looks good!

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 1]
```diff
@@ -24,10 +24,10 @@
 public class Intake extends SubsystemBase {
   private final DoubleSolenoid topSolenoid;
   private final DoubleSolenoid bottomSolenoid;
-  private final CANSparkMax intakeMotor;
+  private final VictorSPX intakeMotor;
 
   /**
-   * Constructs an Intake with a {@link CANSparkMax} at the given CAN ID and a
+   * Constructs an Intake with a {@link VictorSPX} with the given CAN ID and a
```
Comment (chit-sudeep) [inline src/main/java/frc/robot/subsystems/Intake.java:30]: Remove “CAN ID”

**Review by chit-sudeep — CHANGES_REQUESTED** [review]
Fix javadoc

**Review by chit-sudeep — APPROVED** [review]
Sorry for the confusion; just learned that the id the VictorSPXs have is a CAN ID.

**Review by aidan-mundy — APPROVED** [review]

**Pull request merged**